backend/utils/message_utils.py

```diff
@@ -153,3 +153,21 @@ async def update_message(
     message["edited"] = True
 
     return await db.update(settings.MESSAGE_COLLECTION, message_id, message)
+
+async def update_message_threads(
```

Comment:
what's this for and how is it different from `update_threads_message` in threads.py?

Reply:
this one updates threads lists in the message and appends to  it a new message document

it is exactly like the update_messages function but only update the but only appends to the thread